.github/workflows/test.yaml

```diff
@@ -1087,26 +1087,34 @@ jobs:
           - nodejs-postgresql-logs
           - nodejs-mongodb-logs
           - nodejs-tracing
-        clientEngine: ['<dataproxy>'] # value for script logic
+        clientEngine: [<dataproxy>] # value for script logic
         os: [ubuntu-20.04] #, windows-latest, macos-latest]
     runs-on: ${{ matrix.os }}
-    concurrency: dataproxy-${{ matrix.platform }}-${{ matrix.clientEngine }}
+    concurrency:
+      group: dataproxy-${{ matrix.platform }}-${{ matrix.clientEngine }}
```

Comment:
I think this actually needs the flavor so jobs in the same workflow run do not cancel each other, but also across separate runs.